template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/.template.config/template.json

```diff
@@ -38,13 +38,15 @@
         {
           "condition": "(!OrganizationalAuth)",
           "exclude": [
-            "AzureAd/**"
+            "Extensions/AzureAdAuthenticationBuilderExtensions.cs",
```

Comment:
We should also update the namespace for these files to be Microsoft.AspNetCore.Authentication, so they match the namespace of the type the extension method is off of.